lib/circuitry/config/shared_settings.rb

```diff
@@ -0,0 +1,39 @@
+require 'logger'
+
+module Circuitry
+  class ConfigError < StandardError; end
+
+  module Config
+    module SharedSettings
```

Comment:
I know we ran into a road bump with this before, but I forget what.  I thought it was the `attribute` calls.  Oh well, ignore this then.